README.md

```diff
@@ -87,7 +87,7 @@ docker logs docker_aggregator_1
 
 ### Running the python examples
 
-The examples are under [`./python/client_examples`](./python/client_examples).
```

Comment:
Why removing the backticks here?

Reply:
Also, will the links point to something valid once the doc is uploaded on RTD?

Reply:
The sphinx extension `m2r` can't convert this probably. It will look like this 
![Screenshot 2020-04-06 at 11 29 51](https://user-images.githubusercontent.com/10488408/78544418-6a780000-77fa-11ea-8456-50f3103704a7.png)

Good point, I need to check that